cluster/container.go

```diff
@@ -10,3 +16,50 @@ type Container struct {
 	Info   dockerclient.ContainerInfo
 	Engine *Engine
 }
+
+// Status returns a human-readable description of the state
+// Stoken from docker/docker/daemon/state.go
+func (c *Container) Status() string {
+	s := c.Info.State
+	if s.Running {
+		if s.Paused {
+			return fmt.Sprintf("Up %s (Paused)", units.HumanDuration(time.Now().UTC().Sub(s.StartedAt)))
+		}
+		if s.Restarting {
+			return fmt.Sprintf("Restarting (%d) %s ago", s.ExitCode, units.HumanDuration(time.Now().UTC().Sub(s.FinishedAt)))
+		}
+
+		return fmt.Sprintf("Up %s", units.HumanDuration(time.Now().UTC().Sub(s.StartedAt)))
+	}
+
+	if s.Dead {
+		return "Dead"
+	}
+
+	if s.FinishedAt.IsZero() {
+		return ""
+	}
+
+	return fmt.Sprintf("Exited (%d) %s ago", s.ExitCode, units.HumanDuration(time.Now().UTC().Sub(s.FinishedAt)))
+}
+
+// StateString returns a single string to describe state
+// Stoken from docker/docker/daemon/state.go
+func (c *Container) StateString() string {
```

Comment:
@vieux True! Just checked the latest API version and it's not there. This is still in `dockerclient` though so we should probably delete it now ;-)